packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -18,7 +18,7 @@ let _urlResolver: HTMLAnchorElement | undefined;
 
 export function isRelativeUrl(url: string): boolean {
   // A URL is relative if it has no protocol.
-  return !!url && !/^[a-z0-9+-.]:\/\//i.test(url);
+  return !!url && !/^[a-z0-9+-.]+:\/\//i.test(url);
```

Comment:
Is `.` valid in protocols?

Reply:
Good question! It makes me question if any of those characters are actually permitted:

![image](https://user-images.githubusercontent.com/7798177/73105338-8fbcc980-3ead-11ea-9d42-086834e55a87.png)